doc/website/advanced/available-bindings-extensions.md

```diff
@@ -0,0 +1,9 @@
+# Available bindings and extensions
+
+The following bindings and extensions are available for iceoryx:
+
+| Name                | Description                                                                                                                 | Link                                                                                                                 |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| DDS gateway         | Gateway to the DDS protocol using [Cyclone DDS](https://cyclonedds.io/) or [FastDDS](https://github.com/eProsima/Fast-DDS)  | [https://github.com/eclipse-iceoryx/iceoryx-gateway-dds](https://github.com/eclipse-iceoryx/iceoryx-gateway-dds)        |
+| Rust binding        | Binding to use the C++ version of iceoryx in Rust projects                                                                  | [https://github.com/eclipse-iceoryx/iceoryx-rs](https://github.com/eclipse-iceoryx/iceoryx-rs)                         |
+| ara::com binding    | Binding to use iceoryx in [AUTOSAR Adaptive](https://www.autosar.org/standards/adaptive-platform/) projects                                                                         | [https://github.com/eclipse-iceoryx/iceoryx-automotive-soa](https://github.com/eclipse-iceoryx/iceoryx-automotive-soa) |
```

Comment:
@mossmaurice As this is not the real ara::com API, I would use a wording similar to the one used in the repository. E.g. Automoive SOA binding - Example binding for SOA APIs like AUTOSAR Adaptive